src/index.html

```diff
@@ -3,21 +3,18 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Page title</title>
+    <title>Event Booster</title>
     <link rel="stylesheet" href="./sass/index.scss" />
     <script type="module" src="./index.js"></script>
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
   </head>
   <body>
-    <div class="events">
-      aaa
-    </div>
+    <!-- ❗️❗️❗️ Add partials like so for every page -->
+    <include src="./partials/header.html"></include>
+    <div class="events"></div>
     <div class="events__id">
       id
     </div>
-    <!-- ❗️❗️❗️ Add partials like so for every page -->
-
-
-    <p>Adrianna</p>
-    <!-- <include src="./partials/example.html"></include> -->
+    <include src="./partials/example.html"></include>
```

Comment:
to include daj w komentarz żeby się nie pojawiało i można puszczać :)

Reply:
Naprawione :)